### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by fchaillou**
Hello,
While trying to apply the fix using the current readme it failed so here is the updated readme.

Thanks for providing the scalafix rule, a great time gain for that !

### CODE REVIEW

File: 3.0.x/README.md [hunk 1]
```diff
@@ -10,13 +10,13 @@ To use this plugin, please follows these steps (for Maven, use the <a href="http
   - Add `sbt-scalafix` to your `project/plugins.sbt`:
 
 ```
-addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.4")
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.11")
```
Comment (bjaglin) [inline 3.0.x/README.md:13]:
```suggestion
addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.17")
```

File: 3.1.x/README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ To use this plugin with sbt, please follows these steps (for Maven, use the <a h
   - Add `sbt-scalafix` to your `project/plugins.sbt`:
 
 ```
-addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.4")
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.11")
```
Comment (bjaglin) [inline 3.1.x/README.md:13]:
```suggestion
addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.17")
```

**Review by bjaglin — APPROVED** [review]
@cheeseng could we get that in? with the breaking changes in 3.2.0, people are increasingly turning to this but have troubles getting it to work because of the outdated documentation. Thanks!

**Comment by cheeseng** [comment]
@fchaillou Sorry for late response, have been tied up with work recently.  The changes looks good to me, and I think it will be really helpful if we can merge this.

By any chance have you signed before Artima Inc. CLA:

http://www.artima.com/cla/contributorsLicenseAgreement.pdf

or for Corporate:

http://www.artima.com/cla/corpContributorsLicenseAgreement.pdf

Please kindly print, sign and scan back a copy to cla@artima.com, we apologize for any inconvenience caused, we intended to improve the process but somehow it just didn't happen yet.  Good news is it is getting near, though.

**Comment by fchaillou** [comment]
@cheeseng Sorry i don't have a working printer to print the cla.
Feel free to extract a patch and commit it directly.

**Comment by cheeseng** [comment]
@fchaillou Our CLA automation is almost ready, you may be the first to sign our automated CLA soon.  :)

**Comment by cheeseng** [comment]
@cla-bot[bot] check

**Comment by cheeseng** [comment]
@fchaillou Fyi we have our CLA signing up and you can now sign electronically.

Thanks!

**Comment by artimasites** [comment]
@cla-bot[bot] check

**Comment by fchaillou** [comment]
@cheeseng you should be able to merge this now :)

**Pull request merged**